src-built-in/components/myApps/src/components/AddNewAppForm.jsx

```diff
@@ -15,7 +15,7 @@ const VALIDATION_DEFAULTS = {
 	url: true
 };
 
-const INVALID_NAME_MSG = "Name cannot contain special characters or be empty.";
+const INVALID_NAME_MSG = "Name cannot be empty or be a duplicate of an existing component.";
```

Comment:
Is it no longer true that a name cannot contain special characters?  At first glance I don't see anything that changes that functionality.

Reply:
It just hadn't worked previously, so we fixed the message.